### PR TITLE
LPD-40084 Ensure all entities linked to currencies are linked using the currencyCode and not the currencyId (CommercePriceList)

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-price-list-service/service.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@
 
 		<!-- Other fields -->
 
-		<column name="commerceCurrencyId" type="long" />
+		<column name="commerceCurrencyCode" type="String" />
 		<column name="parentCommercePriceListId" type="long" />
 		<column name="catalogBasePriceList" type="boolean" />
 		<column name="netPrice" type="boolean" />
```
Comment (CrescenzoRegaLF) [inline modules/apps/commerce/commerce-price-list-service/service.xml:110]: I fear BChan might have something to say about the lack of alphabetical order

Comment (luca-pellizzon) [reply modules/apps/commerce/commerce-price-list-service/service.xml:110]: It is sorted. The difference is that the first 2 items are external references to other entries, all others are "normal" peoperties

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:upgrade

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**